docs/static/libs/algolia/algolia-mobile.css

```diff
@@ -45,14 +46,13 @@
   }
 }
 
-/* !important tags, box-shadow are needed for desktop, but make overwritting difficult, so moved out of algolia.min.css */
+/* !important tags are needed for desktop but are difficult to overwrite, so moved out of algolia.min.css */
```

Comment:
I don't think this comment is needed, especially with the one that's at the top of this file. Also, does this actually move something out of **algolia.min.css**? Doesn't it override it instead?

Reply:
Yes I moved some position styling out of **algolia.min.css** because it uses **!important** (and needs to in order to work on desktop). 

My mobile media queries didn't seem to override it (even when they themselves had **!important**). So I moved those **!important** position styles here under a desktop media query -- happy to talk next week about ways to work on this!!

Reply:
This sounds tricky! I wonder how we created algolia.min.css. If someone were to generate it/upgrade it to a new algolia version, would it break these changes? If it might, we should search for another approach. Let's talk next week!!